cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -163,7 +181,9 @@ def index_item(item, skip_index=False):
                 item_index['id'] = item_id
                 if item.start:
                     item_index['start_date'] = item.start
-
+                item_index['content_groups'] = getattr(item, 'content_groups', None)
+                if item_index['content_groups'] == []:
+                    item_index['content_groups'] = None
```

Comment:
I don't think this line is even necessary - `[]` is falsy in python, so it shouldn't matter if it is `[]` or `None` if a simple `if` statement is used.